tests/semantic_analyser.cpp

```diff
@@ -2670,14 +2670,13 @@ TEST_F(SemanticAnalyserTest, map_casts_are_global)
 TEST_F(SemanticAnalyserTest, cast_unknown_type)
 {
   test("begin { (struct faketype *)cpu }", Error{ R"(
-stdin:1:9-29: ERROR: Cannot resolve unknown type "struct faketype"
```

Comment:
wait, what happened here? Why is this removed?

Reply:
Yeah, just a mistake. Not sure how it got lost.